lib/cli/watch-run.js

```diff
@@ -114,6 +121,15 @@ const createRerunner = (mocha, beforeRun) => {
   };
 
   const rerun = () => {
+    if (defaultGlobals) {
+      Object.keys(global).forEach(k => {
+        if (defaultGlobals.indexOf(k) === -1) {
```

Comment:
use `if (~defaultGlobals.indexOf(k)) {` like https://github.com/mochajs/mocha/blob/master/lib/runner.js#L215 .
same code style is better.

Reply:
ok. I got it.